packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -22,6 +22,9 @@ export interface Props extends HTMLAttributes<HTMLDivElement> {
   buttonContent?: React.ReactNode | string;
   bottomSpacing?: number;
   topSpacing?: number;
+  additionalContent?: React.ReactNode;
+  /** data source config page design has a narrower icon section */
+  useNewErrorIcon?: boolean;
```

Comment:
please remove, we do not want to redesign this component with this change, must be the figma design that is mistaken